resources/js/components/file-pond.js

```diff
@@ -0,0 +1,91 @@
+import FilePondPluginImagePreview from 'filepond-plugin-image-preview';
+import {create, registerPlugin, setOptions} from 'filepond';
+import de_DE from 'filepond/locale/de-de.js';
+
+export default function ($wire, $ref, lang) {
+    return {
+        tempFilesId: [],
+        isLoadingFiles: [],
+        selectedCollection: null,
+        pond: null,
+        setCollection(collectionName) {
+            this.selectedCollection = collectionName;
+        },
+        loadFilePond() {
+            registerPlugin(FilePondPluginImagePreview);
+
+            const inputElement = $ref.querySelector('#filepond-drop');
+            if (!inputElement) {
+                return;
+            }
+            this.pond = create(inputElement, {
+                allowMultiple: true,
```

Comment:
you should check if the collection accepts multiple or is single file somehow

Reply:
if folder containes a single file - and is constrained to one file - i shoud than prevent file upload ?

Reply:
Warn that the file will be replaced and set allowMultiple to false

Reply:
- on uplaod - signgle file folder - file is replaced (I oversaw the watnign window  - will include it tommorow)
- allowMultiple is updated dynamicaly on folder change 
- if folder is single file - the add folder buttton gets hidden.

Reply:
warning widnow is added - only problem is - Alpine js is throwing the error on rendeLevel(level) -> renderLevel is not defined - although the ui works - will investigate tomorrow